pages/about/index.vue

```diff
@@ -8,8 +8,8 @@
 </template>
 
 <script>
-import OfficerList from "~/components/OfficerList.vue"
-import OrganizationInfo from "~/components/OrganizationInfo.vue"
+import OfficerList from "~/components/about/OfficerList.vue"
```

Comment:
コンポーネントもパスは ~ ではなく @ で統一しましょう！
```suggestion
import OfficerList from "@/components/about/OfficerList.vue"
```